v0/destinations/am/utils.js

```diff
@@ -55,12 +55,19 @@ function getDeviceManufacturer(payload, sourceKey) {
 }
 
 function getPlatform(payload, sourceKey) {
-  const payloadVal = get(payload, sourceKey);
-  return payload.channel
-    ? payload.channel.toLowerCase() === "web"
-      ? "Web"
-      : payloadVal
-    : payloadVal;
+  let payloadVal = get(payload, sourceKey);
+  if (!payloadVal) {
+    payloadVal = "Server";
```

Comment:
As discussed in standup, only in case of warehouse sources, Isn't payloadVal expected to be null ? In that case, can we make it to server ?
Instead `get` logic should be updated to detect the correct server source types

Reply:
How can we verify now if it is coming from warehouse sources?

Reply:
I mean shouldn't the change be done in server SDK's to send the channel as `server`. So only case remaining of payloadVal coming as null will be of warehouse source.
This fix though works only for Acorns provided they don't have any rETL sources